test/integration/targets/eos_interfaces/tests/cli/deleted.yaml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
For integration tests, we should also be asserting the commands sent out.

Reply:
This is... non-trivial. zuul and my test lab have different sets of interfaces, which should not matter to the tests. These tests should pass on any image as long as it has Ethernet1 and Ethernet2 available (and, ideally, using any other set of arbitrary non-management interfaces). Asserting commands is bad enough when we can't guarantee order (well, we can, but it really shouldn't be necessary), and making the check a one-way difference seems like it isn't really testing anything about what the commands are returning.

Other modules should not have this problem, but l2 and l3 will be similarly hamstrung.